src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -332,22 +332,18 @@ export default class ArrayField extends React.Component {
               );
             }
             return (
-              <div key={index} className="va-growable-background">
+              <div key={index} className="va-growable-background editable-row">
```

Comment:
the html here was unnecessarily nested. I wanted to let the extra content occupy the full row, so I removed the columned approach and then positioned the edit button with the new class here and below. 

will this break anything else unknowingly? as far as I can tell, it should preserve the original styling for other situations, but I'd like to double check in other forms using them.

Reply:
Have you checked to see if this behaves as expected on mobile/smaller screens? Besides that I don't see an obvious way that this would break the original styling

Reply:
double checked responsiveness and it appears to be fine. gonna merge